gtda/mapper/tests/test_nerve.py

```diff
@@ -12,13 +12,22 @@
 from gtda.mapper import Projection, OneDimensionalCover, make_mapper_pipeline
 
 
-@settings(deadline=5000)
-@given(X=arrays(dtype=float, unique=True,
-                elements=floats(allow_nan=False,
-                                allow_infinity=False,
-                                min_value=-1e6,
-                                max_value=1e6),
-                shape=array_shapes(min_dims=2, max_dims=2, min_side=11)))
+mapper_input = arrays(
+    dtype=float, unique=True, elements=floats(
+        allow_nan=False, allow_infinity=False,
+        min_value=-1e6, max_value=1e6
+    ),
+    shape=array_shapes(min_dims=2, max_dims=2, min_side=8, max_side=12)
+)
+
+
+hypothesis_settings = dict(
+    deadline=5000, suppress_health_check=(HealthCheck.data_too_large,)
+)
+
+
+@settings(**hypothesis_settings)
+@given(X=mapper_input)
 def test_node_intersection(X):
     # TODO: Replace pipe and graph by Nerve transformer
     # TODO: Improve the Hypothesis strategy to avoid needing to hardcode the
```

Comment:
@wreise so is this TODO now in fact done?  Why is it no longer an issue?

Reply:
Thanks, I missed it!
I'm sure I must have written it, but I don't understand the TODO. By default, `min_intersection=1` in `Nerve`, so the condition we are trying to test should be verified, regardless of the number of points. I don't know why `n_intervals` is mentioned.
I will think about it.

Reply:
Thanks!

Reply:
This second TODO is shared by many tests and I don't see why it applies. I think it was meant to ensure that each interval of the cover contains at least a datapoint. But that is not necessarily the case even if `min_side>n_intervals=10`, since all points could fall into just the "extreme" sets, no?

Looking at the tested statements, I don't think they should require that all sets in the cover have at least one element. And the tests pass, even when there are less points than `n_intervals`. Imo, we can delete this TODO from all the tests. What do you think, @ulupo ?